google/ggrc-core
# GGRC-6665 Add propose changes functionality to infoPane, infoPage

# Issue description

Hover behavior will be looking the same like for user role fields. At the same time new common component should be introduced which will be responsible for the functionality. Propose icon will be available for all users. Edit functionality for reference url and attachment will be still enabled.

# Steps to test the changes

1. Open Control's info pane.
2. Next fields should be redirected:

Description | yes | text
Effective Date | yes | date
Title | yes | varchar(250)
Kind/Nature | yes | int(11)
Type/Means | yes | int(11)
Frequency | yes | int(11)
Fraud Related | yes | tinyint(1)
Significance | yes | tinyint(1)
Notes | yes | text
State | yes | varchar(250)
Assessment Procedure | yes | text
Principal Assignees | yes |  
Secondary Assignees | yes |  
Admins | yes |  
Other Contacts | yes | 
Control Operators | yes |  
Control Owners | yes |  
Assertions
Categories

**Global custom attributes will be handled via separate PR.**
# Solution description

`general-page-header` component was added to provide check for `isChangeableExternally` flag.
`proposable-control` component was added to provide proposal link based on passed attribute name.
`proposable-attribute` component was added to show common info pane / page's attributes.
`role-attr-names-provider` component was added to provide links for custom roles.

# Sanity checklist




# PR Review checklist
